Allow the photo msg to be none (`undefined`) if there isn't any

steward379 opened the pull request:
- Allow the photo msg to be empty
- Update Optimizing text msg

steward379 committed:
- Allow the photo msg to be none (`undefined`) if there isn't any
- Optimize the line text msg format

Yukaii commented on lib/sendLineNotify.ts:
```diff
@@ -2,7 +2,7 @@ import { got } from "got";
   messageContent: string,
-  msgImg: string,
+  msgImg: string | undefined,
   lineNotifyToken: string | undefined
```
Maybe we should drop the `lineNotifyToken` parameter and use `config.lineToken` directly inside the method

steward379 committed:
Enable to send message without image

- Add a conditional expressions to handle situations with or without image msg.
- Delete all the line token parameter in core logic
- Brief line object property with Es6 syntax sugar from `a:a` to `a`

steward379 committed:
Make line text message clear with partial-supported markdown syntax

Yukaii reviewed (approved):
### **ドーンだYO!!**

![img](https://i.imgur.com/FxlAli4.jpg)

Yukaii commented:
/trunk merge
